docs/API.md

```diff
@@ -297,6 +297,39 @@ gulp.task('sometask', function() {
 });
 ```
 
+### lastRun(taskName, [timeResolution])
+
+Returns a timestamp of the last time the task was run successfully. The time 
```

Comment:
I would recommend using the past participle run not ran as "was" needs to be followed by the infinitive run. The present perfect uses the infinitive run. The past tense is ran. Just my few English thoughts ;).